src/components/OrdersWidget/index.tsx

```diff
@@ -96,16 +102,26 @@ const OrdersWidget: React.FC = () => {
       }
     })
 
+    allPendingOrders.forEach(order => {
+      if (!isOrderActive(order, now)) {
+        filteredOrders.expired.pendingOrders.push(order)
```

Comment:
would `expiredOrders` ever have pending orders though?

Reply:
I though the same, but since we are at it, why not? :D

Reply:
seems like UNOPTIMISED CODE

/nooooooooooooooooooooo

Reply:
I also thought the same :) 

It can happen, it's not likely, but if you say it expires in `15min` and has a bad gas price can happen